features/media-transform.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: MediaStreamTrack Insertable Media Processing using Streams
-description: TODO
+name: MediaStreamTrack insertable media processing using streams
```

This seems to be more common (e.g., on MDN and developer.chrome.com)

```suggestion
name: Insertable streams for MediaStreamTrack
```